docs/weaver_code_anatomy.md

```diff
@@ -5,53 +5,50 @@ sidebar_label: View the generated code
 slug: /generated_yaml
 ---
 
-🚧 This page is very much under construction as of 02/17/2021 as we develop the weaver's capabilities. 🚧
+:::warning
+This page is very much under construction as we develop the weaver's capabilities. These 🚧 emoji notes features that are being actively developed in the Weaver and thus might change
+:::
 
-The code [ALWeaver](https://apps-test.suffolklitlab.org/start/assemblylinewizard/assembly_line/#/1&new_session=1) generates is just a starting point. It uses the [labels and variables you added to your documents](doc_vars_reference.md) to make an interview that uses the AssemblyLine library.
+The code generated by [ALWeaver](https://apps-test.suffolklitlab.org/start/assemblylinewizard/assembly_line/#/1&new_session=1) is just a starting point. It uses the [labels and variables you added to your documents](doc_vars_reference.md) to make an interview that uses questions and features from the AssemblyLine library.
 
-Through that code, you can also see examples of [docassemble](https://docassemble.org) features you can use other places.
+This page breaks down what each of these "blocks" (the sections of text that appear between `---`s)
+do individually. You shouldn't have to think about every block as you develop your interview, but as
+you use more advanced features of [docassemble](https://docassemble.org), knowing what these blocks do
+will help.
 
 ## Include
-Imports the AssemblyLine package and, for now, the MassAccess package too. Gives access to the variables in the packages.
+Imports the AssemblyLine package, and the jurisdiction and organization packages that you checked in the Weaver. By default, the jurisdiction package is ALMassachusetts, and the organization package is  the MassAccess package. 
+As described in the [docassemble documentation](https://docassemble.org/docs/initial.html#include), this incorporates the questions in these YAML files, as if the contents of those files were copy pasted right in this block. 
```

Comment:
```suggestion
As described in the [docassemble documentation](https://docassemble.org/docs/initial.html#include), when you run the interview docassemble acts as if the whole contents of these listed YAML files has been copied and pasted in this exact spot.
```

Reply:
I was a bit confused by this. I'm not sure my version is better. Take it or leave it as you wish. Also, we might just want to leave out our own explanation and just leave the link to the documentation.

Reply:
I reorganized this section a bit, and took some of your suggestions. Let me know what you think.

Reply:
Looking much better and really close!